### PR TITLE
Graph theory, Goldbach conjecture, misc.

### PR DESCRIPTION
**Pull request by avekens**
set.mm main (misc.):
* 3imp231 moved from AS's mathbox
* correction in dv-statement for ~sumvtxdg2size
* convention for order of suffixes enhanced (see also discussion in #2383)

set.mm main (graph theory):
* ~usgr1th , ~finsumvtxdgeven, ~vtxdgoddnumeven added

set.mm AV's mathbox (misc):
* alternate definition e// for negated membership e/ and related theorems added (experiment for the discussion in #2398)
* therorems ~evenltle, ~odd2prm2, ~even3prm2 and ~mogoldbblem for even and odd numbers added

set.mm AV's mathbox (Goldbach's conjecture):
* GoldbachOdd renamed to GoldbachOddW and GoldbachOddALTV renamed to GoldbachOdd (as suggested by GL)
* Header comment of section "Goldbach's conjectures" revised (including a glossary)
* comments and label fragments for theorems about Goldbach's conjecture adjusted
* theorems ~sbgoldbb, ~sgoldbeven3prm, ~sbgoldbm, ~mogoldbb, ~sbgoldbmb, ~sbgoldbo about Goldbach's conjecture added
* bibliographic references added/revised (as suggested by BJ)

mmset.raw.html
* bibliographic references enhanced (as suggested by BJ)

### CODE REVIEW

File: mmset.raw.html
Comment (benjub) [inline mmset.raw.html:5791]: I would have linked to `/home.html`, since one can go from there to the pdf, but the other way around in harder (eventhough there's an embedded link in the pdf), but that's a detail. For Helfgott's page, I haven't checked that the syntax is `~~` but we'll see how it diplays.

File: set.mm
Comment (benjub) [inline set.mm:21884]: If you haven't done so, you may try, when you have time, the script `scripts/min.cmd` to minimize theorems using `3imp,3imp31` with `3imp231`. (But don't worry if this takes too much time; this will be done at the next global minimization).
Shouldn't the label be `3impl` ? (not a big deal either)

File: set.mm
Comment (benjub) [inline set.mm]: many more (or far more, or a lot more), since "theorem" is countable.

Also, comma after e.g. (and i.e.) (not a strict rule, but better style, and consistent through set.mm).

File: set.mm
Comment (benjub) [inline set.mm:411194]: Do you really mean "pseudograph" here ? What is its relation with the finite simple graph in that sentence ?

**Review by benjub — COMMENTED** [review]
I made a few comments for the parts in Main. I haven't had time to look at the changes in your mathbox, but the requirements before merging are not the same anyway. I'll let others have a look and merge.

File: mmset.raw.html
Comment (avekens) [reply mmset.raw.html:5791]: I will change the URL to /home.html
The ~~ seems to be correct, it is used also in other URLs. A single ~ caused an error during the CI checks...

File: set.mm
Comment (avekens) [reply set.mm:21884]: I did not check for minimization yet, but I can do later.
I did not change the label chosen by AS, but I think it is correct indicating the changed order of conjuncts. Furthermore, ~impl has a conjunction in its hypothesis, therefore ~3impl seems not to be adequate.

File: set.mm
Comment (avekens) [reply set.mm]: I'll replace "much" by "many"..
I'll also add a comma after e.g., because it is American English (in British English it is not used), and we decided to use American English in set.mm.

File: set.mm
Comment (avekens) [reply set.mm:411194]: You are right, "pseudograph" must be replaced by "graph".

File: set.mm
Comment (benjub) [inline set.mm]: So we have something like `e// = ( ( _V X. _V ) \ _E ) ` (see df-eprel) ?

File: set.mm
Comment (benjub) [inline set.mm]: I guess the "size of a pseudograph" is the number of its edges, and a pseudograph is finite iff its sets of edges and vertices are both finite ?

File: set.mm
Comment (benjub) [inline set.mm]: partition.

**Review by benjub — APPROVED** [review]
Good for me, after reviewing the changes in Main, and after a very quick glance through the changes within your mathbox. So if/when that part is moved to Main, make sure to request a review.

**Comment by avekens** [comment]
@benjub thank you for your approval
Regarding you remarks on my mathbox:
* df-nelbr: good hint - I will add a reference to ~df-eprel to the comment, and try to prove e// = ( ( _V X. _V ) \ _E ) 
* size of a pseudograph: yes, the "size of a (pseudo)graph" is the number of its edges. But because of possible multiedges, a pseudograph can be finite (i.e., has a finite number of vertices) and of infinite size (e.g., infinitely many edges between two vertices).
* _Goldbach partion_ -> _Goldbach partition_ : will be corrected

**Comment by benjub** [comment]
> size of a pseudograph: yes, the "size of a (pseudo)graph" is the number of its edges. But because of possible multiedges, a pseudograph can be finite (i.e., has a finite number of vertices) and of infinite size (e.g., infinitely many edges between two vertices).

I think that a pseudograph should be called finite only if all its components (sets of vertices and of edges) are finite. For the weaker notion, I would simply say "pseudograph with finitely many vertices / pseudograph on a finite set of vertices".

**Review by benjub — APPROVED** [review]

**Comment by tirix** [comment]
Looks good to me!

- It's interesting to define `e/` as a relation. This is probably a better definition than the current one.
- Thanks for adding the precision on the order of suffixes.

**Review by tirix — APPROVED** [review]

**Comment by benjub** [comment]
Merged (sorry @avekens for the merge conflict).

**Comment by avekens** [comment]
@benjub No problem, the merge conflict was easy to solve. Thans for your valuable remarks and merging my changes.

**Pull request merged**
